supplemental-ui/css/cards.css

```diff
@@ -153,6 +153,10 @@ pre {
         flex-direction: column; /* stack image and text vertically */
     }
 
+    .doc {
+        padding: 0 1rem 4rem;
```

Comment:
I think that's a bit too much.

I would do a bit less:
```suggestion
        padding: 0 0.5em;
```

<img width="424" height="493" alt="Image" src="https://github.com/user-attachments/assets/6073987c-cd32-443d-8a9d-b434b68a47c7" />

Also the padding at the bottom is not needed or?